Makefile

```diff
@@ -65,6 +65,48 @@ bbgo-slim-darwin-amd64: $(BIN_DIR)
 
 bbgo-slim-darwin: bbgo-slim-darwin-amd64 bbgo-slim-darwin-arm64
 
+# build native bbgo
+bbgo-dnum: static
+	go build -tags web,release,dnum -o $(BIN_DIR)/bbgo ./cmd/bbgo
+
+# build native bbgo (slim version)
+bbgo-slim-dnum:
+	go build -tags release,dnum -o $(BIN_DIR)/$@ ./cmd/bbgo
+
+# build cross-compile linux bbgo
+bbgo-linux-dnum: bbgo-linux-amd64-dnum bbgo-linux-arm64-dnum
```

Comment:
`bbgo-dnum-linux`, make the platform name as the latest suffix

Reply:
done